### ISSUE TITLE
Unable to edit inline lookup schemas

### ISSUE THREAD

**Issue by piftouf** [cosmocode/dokuwiki-plugin-struct#291]
Hi, I created lookup schemas and created an aggregation page for it with my non-admin profile. I checked that this profile has the proper right on this page and on the schema. I am only able to edit inline the datas with my admin profile. I can suppress or create datas but I cannot edit inline the datas with my non-admin profile.

**Comment by cjohnsonuk** [comment]
Like wise.  I'm using ACL with the authAD plugin.  Double clicking the entry in a table created using 
`---- struct lookup ----
schema: departments
----`
 just highlights the text for all users except the super user.  I've added the username and group or just the username or just the group to the bottom of the lookup definition in the config:allowed editors section but nothing changes.

**Comment by helimech** [comment]
Also having this same issue. I have tried adding groups or individual users to the lookup editors to no avail, same with all relevant ACL's only the super user can inline edit the table.

**Comment by annda** [comment]
Does anyone still have those issues with the current release of Dokuwiki and struct?
I couldn't reproduce them in Firefox or Chrome (Sprintdoc and default template).

**Comment by mtkirchner** [comment]
I got this issues again after some edit of ACL, especially some wrong @ALL rights in a subdirectory.

Release 2020-07-29 "Hogfather"

Now inline editing is admin only.

Following https://forum.dokuwiki.org/d/15743-struct-plugin
I enabled edit rights for @all in root und inline editing works again.

As my wiki is actually closed, this is not a good option for a workaround. 

Any ideas would be wellcome.
